app.drey.Damask.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id": "app.drey.Damask",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "43",
+    "sdk": "org.gnome.Sdk",
+    "command": "damask",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/vala/bin",
+        "prepend-ld-library-path": "/usr/lib/sdk/vala/lib",
+        "env": {}
+    },
```

Comment:
```suggestion
```

you do not need these.

Reply:
Oh interesting. They came from the manifest template that Builder uses for new projects.